RM_SUI/RM_SUI/StoreTabBar/Tabs/TabPlaceholderView.swift

```diff
@@ -0,0 +1,19 @@
+// TabPlaceholderView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// TabView Placeholder
+struct TabPlaceholderView: View {
+    // MARK: - Public View
```

Comment:
```suggestion
    // MARK: - Public Property
```
про константу ниже писать не буду, но лучше вынеси